test/integration/query-interface.test.js

```diff
@@ -138,7 +143,7 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
       });
       await this.queryInterface.dropAllTables({ skip: ['skipme'] });
       let tableNames = await this.queryInterface.showAllTables();
-      if (dialect === 'mssql' || dialect === 'mariadb') {
+      if (dialect === 'mssql' || dialect === 'mariadb' || dialect === 'db2') {
```

Comment:
```suggestion
      if (['mssql', 'mariadb', 'db2'].includes(dialect)) {
```

Like you did in line 104

Reply:
Sorry, my bad. Done.